Examples/Run/Common/include/ActsExamples/Options/CommonOptions.hpp

```diff
@@ -41,7 +41,18 @@ void addMaterialOptions(boost::program_options::options_description& opt);
 void addInputOptions(boost::program_options::options_description& opt);
 
 /// Add common output-related options.
-void addOutputOptions(boost::program_options::options_description& opt);
+enum OutputFormat {
+  None = 0,
+  Root = 1,
+  Csv = 2,
+  Obj = 4,
+  Json = 8,
+  Txt = 16,
+  All = Root | Csv | Obj | Json | Txt
+};
+
+void addOutputOptions(boost::program_options::options_description& opt,
+                      int formats = OutputFormat::All);
```

Comment:
This I find a bit dicey.

Reply:
Yeah, it was for API compatibility. I think when I wrote this I was'nt sure if I will go through all files...
I'll remove the default argument.